packages/web/src/content/docs/docs/gitlab.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: GitLab
+description: Use opencode in GitLab issues and merge requests
+---
+
+opencode integrates with your GitLab workflow.
+Mention `@opencode` in a comment, and opencode will execute tasks within your GitLab CI pipeline.
```

Comment:
Do we need to make it clearer here that the handle is configured/determined by the user?

Reply:
Yeah I'll do it.